magpie/register.py

```diff
@@ -359,7 +359,8 @@ def magpie_register_services_with_db_session(services_dict, db_session, push_to_
     for svc_name, svc_values in services_dict.items():
         svc_new_url = os.path.expandvars(svc_values['url'])
         svc_type = svc_values['type']
-        svc_sync_type = svc_values['sync_type']
+
+        svc_sync_type = svc_values.get('sync_type', svc_values['title'])
```

Comment:
Maybe instead of 'title', let's default to None or other value reflecting not-implemented?